cmd/juju/application/refresh.go

```diff
@@ -277,9 +280,11 @@ func (c *refreshCommand) SetFlags(f *gnuflag.FlagSet) {
 	f.BoolVar(&c.Force, "force", false, "Allow a charm to be refreshed which bypasses LXD profile allow list")
 	f.BoolVar(&c.ForceUnits, "force-units", false, "Refresh all units immediately, even if in error state")
 	f.StringVar(&c.channelStr, "channel", "", "Channel to use when getting the charm from Charmhub")
+	// TODO(jam): 2024-10-10 we allow 'force-series' but call it 'ForceBase' internally, but should be --force-base externally
 	f.BoolVar(&c.ForceBase, "force-series", false, "Refresh even if series of deployed applications are not supported by the new charm")
```

Comment:
We're introducing a new behavior for this flag. It appears specific to old charmstore charm logic. 

In this scenario, using force-series with the base flag only lets you change charm revision which supports the given base. Not to a base which no charm revision in the channel supports.